docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -1,111 +1,107 @@
+2. Create a new wallet and save your secret recovery phrase.
     
     ![](https://i.imgur.com/HEOI0kp.jpg)
 
 3. Login to Metamask.
     
     ![](https://i.imgur.com/zFduIV8.jpg)
 
-4. You should already see an account created. As you can see on top right, it should be pointing to Ethereum mainnet.
+4. By default, Metamask connects to the Ethereum Mainnet (dropdown menu in the top right corner).
     
     ![](https://i.imgur.com/gk3TWUd.jpg)
 
-5. You can use the same account for different networks. For Waku we need to connect to the Sepolia test network.
-6. You can switch to a test network by clicking on the drop down menu. Select Sepolia test network for Waku:
+   To publish messages to the Waku Network, you need to connect to the Sepolia test network.
+5. Switch to the Sepolia test network by selecting it from the dropdown menu. Ensure "Show test networks" is enabled.
 
-    ![](https://i.imgur.com/kbuup6y.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/670778eb-8bf0-42a6-8dd7-1dedfabeeb37)
 
-7. Click on Show/hide test networks.
-8. Enable “Show Test Networks".
-    
-    ![](https://i.imgur.com/02x7iSX.jpg)
+   The same account can be used with different networks. Note that the ETH balance is different for each network (each has its own native token).
     
-    ![](https://i.imgur.com/UFsoRHR.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/0a5aa3a7-359c-4f4b-bd12-bad7c4844b34)
 
-9. Close the settings and now you should see the test networks in the drop down on the top right.
+6. To view the private key for this account, click on the three dots next to the account name and select "Account Details".
     
-    ![](https://i.imgur.com/leSYt4z.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/83fffa23-4a3b-46f9-a492-9748bfd47cff)
 
-10. Switch to Sepolia Test Network.
-11. You can see that the same account can be used with different networks. But note that the ETH balance on different networks are different and are different tokens.
+   Select "Show Private Key".
     
-    ![](https://i.imgur.com/Y7YkuEe.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/3a513389-2df1-4e32-86da-a1794126cdac)
 
-12. Now to export your private key for the account, if needed, please click on the three dots beside the account.
+   Enter your Metamask password and click "Confirm"
     
-    ![](https://i.imgur.com/9BlBmhT.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/ffbac631-b933-4292-a2c6-dc445bff153c)
 
-13. Click on Account Details.
-14. Click on Export Private Key.
-    
-    ![](https://i.imgur.com/7g6SyuX.jpg)
+   You will be shown the private key.
+   This private key is used elsewhere by Waku RLN registration tools to assist with membership registration in the Sepolia test network.
+   We strongly recommend that you create an account only for this purpose.
+   NEVER expose a private key that controls any valuable assets or funds.
```

Comment:
Could we move that at the start. Let's not have ppl do this with account with assets.

Reply:
Thanks! Done in https://github.com/waku-org/nwaku/pull/2206/commits/62cc7aa792716e25408c5c670be0ca82c4699a2d